src/commands.ts

```diff
@@ -585,6 +585,50 @@ export class CommandCenter {
 		}
 	}
 
+	@command('hg.purgeFiles', { repository: true })
+	async purgeFiles(repository: Repository): Promise<void> {
+		return this._purge(repository.untrackedGroup.resources);
+	}
+
+	@command('hg.purgeAll', { repository: true })
+	async purgeAll(repository: Repository): Promise<void> {
+		if (await interaction.confirmDeleteUntrackedAndIgnored()) {
+			return await repository.purgeAll();
+		}
+	}
+
+	@command('hg.purge')
+	async purge(...resourceStates: SourceControlResourceState[]): Promise<void> {
+		if (resourceStates.length === 0) {
```

Comment:
Why should there be an option to purge specific files? Selecting specific files to send to the `purge` command is the same thing as selecting specific files and then deleting them.

Reply:
Yes, you're right. Currently there is no way of deleting untracked files from the SCM tab, so this is what I am implementing.